src/components/Icon/MucIcon.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <svg
+    aria-hidden="true"
+    class="m-button__icon"
+    :style="{ color: color }"
+  >
+    <use :xlink:href="'#icon-' + icon" />
```

Comment:
```suggestion
    <use :href="'#icon-' + icon" />
```

xlink-href ist [deprecated](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href?retiredLocale=de)